mkdocs.yml

```diff
@@ -24,27 +24,28 @@ nav:
     - Project Tags: our_work/tags.md
     - Past/Current Projects:
       - Current Projects:
+        - A&E Forecasting Tool: our_work/a_and_e_forecasting_tool.md
+        - AI Assurance Research Path: our_work/ds278_Assurance_Research_Path.md
         - AI Dictionary: our_work/ai-dictionary.md
         - AI Ethics in Practice: our_work/ai-ethics.md
+        - Cancer high-risk cohorts: our_work/cancer_high_risk_cohorts.md
+        - "Corporate Services: Pipeline rebuild": our_work/corporate_services_pipeline_rebuild.md
+        - CVD Pathways: our_work/cvd_pathways.md
         - Data Linkage Hub: our_work/data-linkage-hub/
         - Emerging Privacy Enhancing Technologies: our_work/pets.md
+        - ePMA Auto Coding: our_work/epma_autocoding.md
         - NHS.UK Automatic Moderation of Ratings & Reviews: our_work/ratings-and-reviews.md
+        - Primary Care Data Generator: our_work/swpclab.md
+        - RAG: our_work/ds251_RAG.md
```

Comment:
RAG should be in the past projects! (I realise i moved it to past but never removed it from current, if you wouldnt mind removing it whilst youre here please do :)) If you wont be making any other commits then dont bother ill fix it later